app/partials/main-balance.pug

```diff
@@ -1,24 +1,18 @@
-.heavy.flex-column.flex-align-end.right-align.center-mobile.main-balance.pointer(ng-click="toggleDisplayCurrency()", ng-show="status.didUpgradeToHd")
-  span(ng-show="getTotal() == null")
-    h3
-      img(ng-show="balance == null" src="img/spinner.gif")
-  span.flex-between.flex-column(ng-show="getTotal() != null")
-    .mvn.f-24.em-300.lh1.mb-10.flex-end.flex-justify-mobile
-      .flex-row
-        .flex-row.flex-center
-          i.icon-bitcoin.f-20.mrs
-          span(ng-show="isBitCurrency(settings.displayCurrency)")
-            span {{ getTotal() | toBitCurrency:settings.btcCurrency }}
-          span(ng-hide="isBitCurrency(settings.displayCurrency)")
-            fiat(btc="getTotal()")
-        .flex-row.flex-center(ng-hide="getEthTotal() === null")
-          .vertical-line.mh-15.hidden-sm.hidden-xs
-          i.icon-ethereum.f-20.mrs
-          span(ng-show="isBitCurrency(settings.displayCurrency)")
-            span {{ getEthTotal() | format:'eth' }} ETH
-          span(ng-hide="isBitCurrency(settings.displayCurrency)")
-            fiat(eth="getEthTotal()")
-    .mvn.f-20.em-200.lh1(ng-class="{'security-red flex-end flex-center flex-justify-mobile': balanceMessage}")
-      .fork-warn.mr-10.flex-center.flex-end(ng-show="balanceMessage")
-        span.f-10 {{ balanceMessage }}
-      fiat(btc="getTotal()" eth="getEthTotal() > 0 ? getEthTotal() : null")
+.heavy.flex-column.flex-align-end.right-align.center-mobile.main-balance(ng-show="status.didUpgradeToHd")
+  span.f-20(translate="{{ !activeAsset() ? 'BALANCES.TOTAL' : 'BALANCES.' + activeAsset() }}")
+  .pointer(uib-dropdown)
+    .flex-row.flex-center.f-20.em-200(uib-dropdown-toggle)
+      include ./context-balance
+    ul.dropdown-menu-right.pbn(uib-dropdown-menu)
+      li.flex-row.flex-center.flex-end.f-20.em-200
+        include ./context-balance
+      li.flex-row.flex-end.mb-5(ng-repeat="asset in assets" ui-sref="{{ 'wallet.common.' + asset.code + '.transactions' }}" ng-if="activeAsset() !== asset.code")
+        span.mr-10 {{ asset.total() | convert:asset.code:true }}
+        fiat(btc="asset.code === 'btc' && assets.btc.total()" eth="asset.code === 'eth' && assets.eth.total()" bch="asset.code === 'bch' && assets.bch.total() / 1e8")
+      li.flex-row.flex-end.mb-5(ui-sref="wallet.common.home" ng-if="activeAsset()")
+        span.mr-10 Total Balance
+        fiat.mid-grey(btc="assets.btc.total()" eth="assets.eth.total()" bch="assets.bch.total() / 1e8")
+.mvn.f-20.em-200.lh1(ng-show="balanceMessage" ng-class="{'security-red flex-end flex-center flex-justify-mobile': balanceMessage}")
+  .fork-warn.mr-10.flex-center.flex-end
+    span.f-10 {{ balanceMessage }}
+  fiat(btc="btc.total()" eth="eth.total() > 0 ? getEthTotal() : null" bch="bch.total()")
```

Comment:
This line isn't very consistent with line 14, but they look like they're trying to do the same thing, is this right?
`fiat.mid-grey(btc="assets.btc.total()" eth="assets.eth.total()" bch="assets.bch.total() / 1e8")`